ownCloud/Resources/en.lproj/Localizable.strings

```diff
@@ -267,7 +267,7 @@
 "Deselect All" = "Deselect All";
 "All Photos" = "All Photos";
 "Albums" = "Albums";
-"Importing '%@' from photo library" = "Importing '%@' from photo library";
+"Importing asset from photo library" = "Importing asset from photo library";
```

Comment:
While not wrong, I think `item` may be more familiar to regular users.

Reply:
@felix-schwarz fixed

Reply:
"Importing from photo library" is even better. Thanks!